### PR TITLE
Add nmstate/kubernetes-nmstate branch protection

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/config.yaml [hunk 1]
```diff
@@ -180,6 +180,14 @@ branch-protection:
               protect: true
         kubectl-virt-plugin:
           protect: true
+    nmstate:
+      repos:
+        kubernetes-nmstate:
+          protect: true
+          required_status_checks:
+            contexts:
+              - pull-kubernetes-nmstate-unit-test
```
Comment (dhiller) [inline github/ci/prow/files/config.yaml]: these are not necessary to mention, branchprotection will add them automatically from the presubmits and make them required, if they are in the job definition.

Comment (qinqon) [reply github/ci/prow/files/config.yaml]: so if they are marked at 'required' at job definitions they will bee added right ?

Comment (dhiller) [reply github/ci/prow/files/config.yaml]: Yes, that should work. Please see that these are picked up for kubevirt/kubevirt also automatically. We just added travis and coverage.

**Pull request merged**